tests/run-macros/i19951-java-annotations-tasty-compat/JavaAnnot_1.java

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
Could you add a version of this test without the _X filename suffixes to exercise joint compilation?

Reply:
Well, no I can't do that. The test is precisely about a binary compatible *evolution* of the Java annotation. One file actively erases the output of another one. They cannot all be compiled in one run.

I can merge _1 and _2, and merge _3 and _4. That's the smallest amount of runs that makes sense. Would that address your request?

Reply:
I added a test with only `_1` and `_2`. It did find an issue which I also fixed.

Reply:
Yes indeed, that's what I had in mind.